### PR TITLE
Add 'How Government Works' page

### PR DESCRIPTION
**Pull request by brucebolt**
This adds the [How Government Works](https://www.gov.uk/government/how-government-works) page, which is currently rendered by Whitehall.

| Whitehall | Government Frontend |
| -- | -- |
| Screenshot of heading rendered by Whitehall | Screenshot of heading rendered by Government Frontend |
| Screenshot of ministers section rendered by Whitehall | Screenshot of ministers section rendered by Government Frontend |

[Trello card](https://trello.com/c/lIvye0eU)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

**Comment by JonathanHallam** [comment]
Can't leave a comment in the right place! The images could be moved into the "buildings" folder, would mean we could rename the `how gov works` folder to people. I'm not sure I care that much tbh but it does seem a little odd to have a no.10 pic in a folder other than buildings

**Review by JonathanHallam — APPROVED** [review]
Looking good, thanks :)

**Comment by brucebolt** [comment]
> Can't leave a comment in the right place! The images could be moved into the "buildings" folder, would mean we could rename the `how gov works` folder to people. I'm not sure I care that much tbh but it does seem a little odd to have a no.10 pic in a folder other than buildings

Yeah, I think the whole images directory needs reviewing.  I didn't want to put these into `buildings` are they are nested under `history`, which isn't really what this page is about.



> Looking good, thanks :) Could you add a couple of images to the PR description please for the sake of comparison

Done

File: app/views/content_items/how_government_works.html.erb [hunk 1]
```diff
@@ -141,65 +141,37 @@
...
-                <%= render "govuk_publishing_components/components/big_number", {
-                  number: @content_item.ministerial_role_counts["prime_minister"],
-                } %>
-
-                <span class="govuk-visually-hidden">&nbsp;</span>
-                <span class="govuk-visually-hidden">
-                  Prime Minister
-                </span>
-              </a>
+              <%= render "govuk_publishing_components/components/big_number", {
```
Comment (hannako) [inline app/views/content_items/how_government_works.html.erb:144]: Oh that's nicer, all those `<span class="govuk-visually-hidden">&nbsp;</span` were a bit :grimacing:

File: app/views/content_items/how_government_works.html.erb [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+                <div class="depts__item govuk-!-display-inline-block">
+                  <%= render "govuk_publishing_components/components/big_number", {
+                    number: @content_item.department_counts["non_ministerial_departments"],
+                    label: "Non-ministerial departments",
+                    href: "/government/organisations#non_ministerial_departments"
+                  } %>
+                </div>
+                <div class="depts__item govuk-!-display-inline-block">
+                  <%= render "govuk_publishing_components/components/big_number", {
+                    number: "#{@content_item.department_counts["agencies_and_public_bodies"].floor(-2)}+",
```
Comment (hannako) [inline app/views/content_items/how_government_works.html.erb:263]: Could this go into the presenter too?

File: app/views/content_items/how_government_works.html.erb [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+
+      <section>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "History of government",
+          id: "history-uk-government",
+          font_size: "l",
+          margin_bottom: 4,
+        } %>
+
+        <p class="govuk-body">Britain has one of the oldest governments in the world. Find out more about how it has worked and who has shaped it in the <%= link_to "history", "/government/history/", class: "govuk-link" %> section.</p>
```
Comment (hannako) [inline app/views/content_items/how_government_works.html.erb:516]: the href `/government/history` has a rogue `/` at the end of it. I know it still works, but whilst you're in the area!

**Review by hannako — APPROVED** [review]
Looks good Bruce. Just a few minor comments.

File: app/views/content_items/how_government_works.html.erb [hunk 4]
```diff
@@ -0,0 +1,552 @@
...
+          <% if @content_item.prime_minister.present? %>
+            <p class="govuk-body">The Prime Minister is <%= link_to @content_item.prime_minister["title"], @content_item.prime_minister["base_path"], class: "govuk-link" %>.</p>
+          <% end %>
+
+          <p class="govuk-body govuk-!-margin-bottom-8">
+            <%= link_to "Read more about the Prime Minister's Office, 10 Downing Street", "/government/organisations/prime-ministers-office-10-downing-street", class: "govuk-link" %>
+          </p>
+
+          <% if @content_item.prime_minister.present? && @content_item.prime_minister.dig("details", "image").present? %>
+            <%= image_tag @content_item.prime_minister.dig("details", "image", "url"), { alt: @content_item.prime_minister.dig("details", "image", "alt_text"), width: '465', height: '310' } %>
```
[the remark this answers is not in the document]
Comment (brucebolt) [reply app/views/content_items/how_government_works.html.erb:106]: Have moved the image URL and alt text into presenter methods

File: app/views/content_items/how_government_works.html.erb [hunk 5]
```diff
@@ -0,0 +1,552 @@
...
+    <div class="govuk-grid-column-two-thirds">
+      <section class="govuk-!-margin-bottom-9">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Ministers",
+          font_size: "m",
+          heading_level: 3,
+          margin_bottom: 4,
+        } %>
+
+        <% if !@content_item.reshuffle_in_progress? %>
```
[the remark this answers is not in the document]
Comment (brucebolt) [reply app/views/content_items/how_government_works.html.erb:141]: We don't lint views, so rubocop didn't pick this up and it was copied straight over from Whitehall.  I'll change to use `unless` as it's clearer.

File: app/views/content_items/how_government_works.html.erb (hunk 1 above)
Comment (brucebolt) [reply app/views/content_items/how_government_works.html.erb:144]: Yeah, I think it was so the label would display above the number but a screenreader would read the label after the number.  The big number component solves this by always putting the label after the number.

File: app/views/content_items/how_government_works.html.erb (hunk 2 above)
Comment (brucebolt) [reply app/views/content_items/how_government_works.html.erb:263]: I'll move the logic for this into the presenter.

File: app/views/content_items/how_government_works.html.erb (hunk 3 above)
Comment (brucebolt) [reply app/views/content_items/how_government_works.html.erb:516]: Cool, I'll remove this (more copy-and-paste from Whitehall inconsistencies!)

File: app/views/content_items/how_government_works.html.erb [hunk 6]
```diff
@@ -0,0 +1,552 @@
...
+                <span class="govuk-visually-hidden">
+                  Prime Minister
+                </span>
+              </a>
+            </div>
+
+            <div class="govuk-grid-column-one-half">
+              <p class="govuk-!-margin-bottom-1 govuk-heading-s">Cabinet ministers</p>
+
+              <a class="govuk-link gem-c-big-number__link cabinet-ministers" href="/government/ministers#cabinet-ministers">
```
[the remark this answers is not in the document]
Comment (brucebolt) [reply app/views/content_items/how_government_works.html.erb]: I've replaced the remaining links to all use `link_to`.

**Pull request merged**